lib/engine/game/g_1848/step/dividend.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/dividend'
+
+module Engine
+  module Game
+    module G1848
+      module Step
+        class Dividend < Engine::Step::Dividend
+          BOE_BASE_PAYOUT = { '2' => 0, '3' => 100, '4' => 100, '5' => 200, '6' => 200, '8' => 300 }.freeze
+
+          def actions(entity)
+            return super unless entity == @game.boe
+
+            # boe always pays if it has any revenue
+            process_dividend(Action::Dividend.new(entity, kind: 'payout'))
+            []
+          end
+
+          def corporation_dividends(_entity, _per_share)
+            0
+          end
+
+          def change_share_price(entity, payout)
+            return super unless entity == @game.boe
+          end
+
+          def dividend_options(entity)
+            return super unless entity == @game.boe
+
+            revenue = calculate_boe_revenue
```

Comment:
Do you have dividend options for BOE? It always pays out. I think you can skip the options step.

Reply:
I think 1840 does this.